server.js

```diff
@@ -7,23 +7,155 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
+// const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/people"
+// mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
+// mongoose.Promise = Promise
+
+// const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/notes"
+// mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
+// mongoose.Promise = Promise
+
+// const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/post-codealong"
+// mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
+// mongoose.Promise = Promise
+
+// //Create Note model with Damian
+// const Note = mongoose.model('Note', {
+//   text: String,
+//   createdAt: {
+//     type: Date,
+//     default: () => new Date()
+//   }
+// })
+
+//Create a model for mongo database as of VAN
+// const Person = mongoose.model('Person', {
+//   name: {
+//     type: String,
+//     required: true,
+//     minlength: 2,
+//     minheight: 500
+//   },
+//   height: {
+//     type: Number,
+//     required: true,
+//     min: 5
+//   },
+//   birthdate: {
+//     type: Date,
+//     default: Date.now
+//   }
+// })
+
+// new Person({ name: "Nina", height: 150 }).save()
+
+
+//Code-along with Task by VAN 
+// const Task = mongoose.model('Task', {
+//   text: {
+//     type: String,
+//     required: true,
+//     minlength: 5
+//   },
+//   complete: {
+//     type: Boolean,
+//     default: false
+//   },
+//   createdAt: {
+//     type: Date,
+//     default: Date.now
+//   }
+// });
+
+// Message for Happy Thought, validation of 5-140 characters, hearts and created date.
+const Thought = mongoose.model('Thought', {
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140
+  },
+
+  hearts: {
+    type: Number,
+    default: 0
+  },
+
+  createdAt: {
+    type: Date,
+    default: () => new Date
+  }
+})
+
+
+
 // Defines the port the app will run on. Defaults to 8080, but can be 
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
+//Middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+// ROUTES
```

Comment:
Also add the error handling for the database connection. Check Damiens video about setting that middleware up here:
https://technigo.wistia.com/medias/l5wqrcu4to?wtime=9m22s

Reply:
I don't get what you are asking for, Jennie? Can please clarify?